### PR TITLE
Clean up BuildFiles under CalibCalorimetry/

### PR DESCRIPTION
**Pull request by fwyzard**
Clean up BuildFile.xml filea with `xmllint --format`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by fwyzard** [comment]
@cmsbuild, please test

File: CalibCalorimetry/EcalCorrelatedNoiseAnalysisModules/BuildFile.xml [hunk 1]
```diff
@@ -1,11 +1,8 @@
-<use   name="FWCore/Framework"/>
-<use   name="FWCore/ParameterSet"/>
-<use   name="DataFormats/EcalDigi"/>
-<use   name="DataFormats/EcalDetId"/>
-<use   name="TBDataFormats/EcalTBObjects"/>
-<use   name="CalibCalorimetry/EcalCorrelatedNoiseAnalysisAlgos"/>
-<use   name="root"/>
-<flags   EDM_PLUGIN="1"/>
-#<export>
```
Comment (silviodonato) [inline CalibCalorimetry/EcalCorrelatedNoiseAnalysisModules/BuildFile.xml:9]: @fwyzard

File: CalibCalorimetry/EcalTPGTools/test/BuildFile.xml [hunk 2]
```diff
@@ -1,5 +1,3 @@
-<use   name="CalibCalorimetry/EcalTPGTools"/>
-<use   name="OnlineDB/EcalCondDB"/>
-<use   name="OnlineDB/Oracle"/>
-#<bin   file="changeTPGLutThreshold.cpp">
```
Comment (silviodonato) [inline CalibCalorimetry/EcalTPGTools/test/BuildFile.xml:4]: @fwyzard

Comment (fwyzard) [reply CalibCalorimetry/EcalTPGTools/test/BuildFile.xml:4]: this was commented out, I removed it

File: CalibCalorimetry/EcalCorrelatedNoiseAnalysisModules/BuildFile.xml (hunk 1 above)
Comment (fwyzard) [reply CalibCalorimetry/EcalCorrelatedNoiseAnalysisModules/BuildFile.xml:9]: this was commented out, I removed it

**Comment by silviodonato** [comment]
merge

**Pull request merged**